russstation/code/game/turfs/closed/mineral_walls.dm

```diff
@@ -0,0 +1,74 @@
+/turf/closed/wall/mineral/stone
+	name = "stone wall"
+	desc = "A wall made of stone, designed to stonewall trespassers."
+	icon = 'russstation/icons/turf/walls/stone.dmi'
+	icon_state = "wall-0"
+	base_icon_state = "wall"
+	sheet_type = /obj/item/stack/sheet/mineral/stone
+	hardness = 45 // between iron walls and wood walls
+	explosion_block = 0
+	smoothing_flags = SMOOTH_BITMASK
+	smoothing_groups = list(SMOOTH_GROUP_CLOSED_TURFS, SMOOTH_GROUP_WALLS, SMOOTH_GROUP_SANDSTONE_WALLS)
+	canSmoothWith = list(SMOOTH_GROUP_SANDSTONE_WALLS) // pretty much the same thing aesthetically
+	initial_gas_mix = LAVALAND_DEFAULT_ATMOS
+	custom_materials = list(/datum/material/stone = 4000)
+	girder_type = /obj/structure/girder/stone
+	decon_type = /turf/open/floor/stone/russ
+
+/turf/closed/wall/mineral/stone/deconstruction_hints(mob/user)
+	return span_notice("The outer stones are firmly attached by <b>Dwarven engineering</b>.")
+
+/turf/closed/wall/mineral/stone/attackby(obj/item/I, mob/user, params)
+	// copied from parent, but removed advanced tool check
+	user.changeNext_move(CLICK_CD_MELEE)
+
+	//get the user's location
+	if(!isturf(user.loc))
+		return //can't do this stuff whilst inside objects and such
+
+	add_fingerprint(user)
+
+	var/turf/T = user.loc //get user's location for delay checks
+
+	//the istype cascade has been spread among various procs for easy overriding
+	// no wall mounting for dorfs right now
+	if(try_clean(I, user, T) || try_decon(I, user, T))
+		return
+
+	return // no parent call
```

Comment:
I know why this doesn't call parent but the fact that it doesn't seems like a bad idea (attackby is a signal proc)